modules/formulize/language/english/main.php

```diff
@@ -240,6 +240,9 @@
 define("_formulize_SAVEREPORTBUTTON", "Save");
 define("_formulize_REPORTNAME", "Report Name:");
 define("_formulize_ANDORTITLE", "Interfield AND/OR Setting:");
+define("_formulize_DAYS", "Days");
```

Comment:
Because I'm a dick, I'm prepared to live without a complete French translation of the admin UI (mostly because it hasn't been complete for a very, very long time if ever). But the front side stuff needs a proper French translation, I consider that a minimum requirement, anything end user-facing needs English / French because we're in a Canada and if we are going to get Canadian orgs to use this, gotta be bilingual.

Reply:
Should just be a matter of making a main.php modification in the /french folder. But note the issue above in the render method about using these for the captions of the boxes.